testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/util/IcebergTestUtils.java

```diff
@@ -26,6 +26,11 @@ public static String getTableLocation(String tableName)
         return (String) onTrino().executeQuery("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*/[^/]*$', '') FROM " + tableName).getOnlyValue();
     }
 
+    public static String getLatestMetadataLocation(String catalog, String schema, String tableName)
```

Comment:
This method returns metadata file name, right? I would rename to `getLatestMetadataFileName` or something.

Reply:
Done!